services/port_manager/src/main/resources/application.properties

```diff
@@ -8,7 +8,7 @@ microservices.ip.service.url=http://localhost:9004/ips
 microservices.mac.service.url=http://localhost:9005/macs
 microservices.sg.service.url=http://localhost:9008
 microservices.node.service.url=http://localhost:9007/nodes
-microservices.dataplane.service.url=http://localhost:9010/v4/port/
+microservices.dataplane.service.url=http://localhost:9010/network-configuration
```

Comment:
@chenpiaoping Please also update the yaml file
https://github.com/futurewei-cloud/alcor/blob/28cfda5157c0f5e3bfaa4330c6b536217230b5d2/kubernetes/services/port_manager.yaml#L23

Reply:
> microservices.dataplane.service.url=http://dataplanemanager-service.default.svc.cluster.local:9010/v4/port/

OK, I will do that.